charts/kube-prometheus-stack/hack/sync_prometheus_rules.py

```diff
@@ -107,7 +107,15 @@ def new_representer(dumper, data):
         'destination': '../templates/prometheus/rules-1.14',
         'min_kubernetes': '1.14.0-0',
         'mixin': """
-        local kp = { prometheusAlerts+:: {}, prometheusRules+:: {}} + (import "mixin.libsonnet");
+        local kp =
+            { prometheusAlerts+:: {}, prometheusRules+:: {}} +
+            (import "mixin.libsonnet") +
+            {'_config': {
+                'etcd_selector': 'job=~".*etcd.*"',
+                'etcd_instance_labels': 'instance, pod',
```

Comment:
Could you leave a reference to https://github.com/etcd-io/etcd/blob/1c22e7b36bc5d8543f1646212f2960f9fe503b8c/contrib/mixin/config.libsonnet#L13

That would be great!